docs/source/challenge-doc.rst

```diff
@@ -119,29 +152,33 @@ the simulator might not be accessible in submissions.
 
     - Hand passing error measures the distance between the MPL and the object
 
-    - The manipulated object has full 6 degrees of freedom, its state described as a 7 dimensional value in position + quaternion format. Details can be found in "`freejoint <https://mujoco.readthedocs.io/en/stable/XMLreference.html#body-freejoint>`__"
+    - The manipulated object has full 6 degrees of freedom, its state described as a 7 dimensional value in position + quaternion format. Details can be found in "`mujoco-freejoint <https://mujoco.readthedocs.io/en/stable/XMLreference.html#body-freejoint>`__" page
 
 
 
 
 **Variation on Object Properties**
-The geometry, mass, and friction of the object will reset at the start of each episode. 
+Both the geometry and physical properties of the object as well as the environment can be sampled at the start of each episode to provide variability in the task. Provided 
+below is an example of how real-world scenarios is captured in the test environments we provide.
 
-    - Object scale: a +- change between 0% - 5%, 0% - 10% scale variations in respective geom directions 
-    - Object Mass: an upper/lower bound of +-50 gms
+    - Object scale: a +- change in respective geom directions ( between 0% - 5%, 0% - 10% in TEST environment)
```

Comment:
do we still have TEST vs EVALUATION environments? 
Anyways, better to point to the environment names

Reply:
ok... I might have missed the description then.. all good :)